enrichment-ner/src/main/java/eu/europeana/enrichment/ner/linking/WikidataServiceImpl.java

```diff
@@ -412,11 +412,15 @@ private String createRequestWikiId(String wikidataId, int... retry) throws Excep
 					return responeString;
 				}
 				else {
-					throw new FunctionalRuntimeException("Wikidata response json is invalid (does not contain \"entities\").");

```

Comment:
For some entities we check like: http://www.wikidata.org/entity/Q12205470, where the entity does not exist, we do not get the 200 code in response, so we do not throw the exception to break the analysis but return null